README.md

```diff
@@ -6,59 +6,8 @@ Types of geometries in Who's On First, and their meaning.
 
 _This document should still be considered a work-in-progress_
 
-## Polygons
```

Comment:
Let's point people to the new README from the general project readme, since otherwise it's not obvious all the good stuff moved.

Reply:
done here:  https://github.com/whosonfirst/whosonfirst-geometries/pull/1/commits/4ea74a2c8f42b0662c6356577be59ecee08ac730